apps/kserve/data.yaml

```diff
@@ -0,0 +1,79 @@
+tags:
+  - AI/Machine Learning
+  - Serverless
+title: KServe
+summary: "Serverless platform for deploying, monitoring, and managing ML models on Kubernetes."
+logo: ./assets/kserve.png
+logo_big: ./assets/kserve_logo.svg
+description: |
+    k0rdent facilitates the deployment and lifecycle management of KServe within Kubernetes clusters by leveraging its multi-cluster orchestration capabilities.
+    KServe provides a Kubernetes-native platform for serving and managing machine learning models with high performance and scalability.
+    
+    KServe builds on Kubernetes to provide a standardized, scalable, and reliable solution for deploying, monitoring, and managing 
+    machine learning models in production. With features like traffic management, canary deployments, and autoscaling, KServe enables 
+    seamless deployment of models across various frameworks including TensorFlow, PyTorch, scikit-learn, XGBoost, and ONNX.
+    
+    KServe consists of two main components:
+    1. **KServe CRDs** - Custom Resource Definitions that provide the foundation for KServe's functionality
+    2. **KServe Controller** - The controller that implements serverless model serving capabilities
+    
+    Through k0rdent, users can easily install and manage KServe across multiple clusters, providing a centralized control plane
+    to monitor health, enforce policies, and enable enterprise-grade machine learning serving. This application allows deploying 
+    the KServe Controller in Serverless mode without ModelMesh functionality enabled, which is optimized for environments 
+    where autoscaling and scale-to-zero capabilities are prioritized.
+    
+    By deploying KServe from the catalog, k0rdent enables users to build and scale ML inference services while leveraging Kubernetes-native infrastructure.
+support_link: https://kserve.github.io/website/master/
```

Comment:
Please use "support_link" only for commercial support contact, otherwise just comment it out.

Reply:
Done.